File: Elixir/abc191_b/lib/main.ex
defmodule Main do
  def main do
    [_ | [x]] = IO.read(:line) |> String.trim() |> String.split(" ")

    IO.read(:line)
    |> String.trim()
    |> String.split(" ")
    |> Enum.reject(fn y -> y == x end)
    |> to_output_format_list()
    |> IO.puts()
  end

  def to_output_format_list(list) do
    list |> Enum.join(" ")
  end
end

File: Clojure/abc191_b/main.clj
(defn get-line
  "Get input data."
  []
  (read-line))

(defn split-words
  "Returns an array of input data split by spaces."
  []
  (clojure.string/split (get-line) #" "))

(defn solve [a x]
  (filter #(not (= % x)) a))

(defn main []
  (let [x (second (split-words))
        a (split-words)]
    (println (clojure.string/join " " (solve a x)))))

(main)